Exercises/arrays-methods/arrays-methods.js

```diff
@@ -0,0 +1,22 @@
+// Create a function - predicate which will return `true` if passed element present in array and `false` - if not.
+
+function includes(array, element) {
+  return array.includes(element);
+}
+
+// Create a function which will check that all array elements are even.
+
+function checkEven(array) {
+  return array.every((element) => element % 2 === 0);
```

Comment:
you can omit comparing with `0`

Reply:
@lizarusi , I dont have any idea how to check that number is even/odd without comparing with 0 :(

Reply:
The result of `% 2` expression is 0 or 1 so it could be converted to boolean without comparion